.github/workflows/cd-syft.yml

```diff
@@ -109,9 +109,25 @@ jobs:
           tox -e syft.build.helm
           tox -e syft.package.helm
 
+      # Check if the version is a pre-release
+      - name: Check if the version is a pre-release and modify version string
+        id: release_checks
+        run: |
+          if [[ $(python packages/grid/VERSION) == *"beta"* ]]; then
+            echo "is_pre_release=true" >> $GITHUB_OUTPUT
+            echo "release_tag=beta" >> $GITHUB_OUTPUT
+            echo "github_release_version=$(python packages/grid/VERSION |  sed 's/-beta./b/')" >> $GITHUB_OUTPUT
+          else
+            echo "is_pre_release=false" >> $GITHUB_OUTPUT
+            echo "release_tag=latest" >> $GITHUB_OUTPUT
+            echo "github_release_version=$(python packages/grid/VERSION)" >> $GITHUB_OUTPUT
+          fi
+
       - name: Check and Bump Protocol Version
         run: |
-          # export BUMP=True
+          if [ "${{ steps.release_checks.outputs.is_pre_release }}" = "false" ]; then
+            export BUMP= True
```

Comment:
NITPICK(my fault): we could remove space between bump and true ,as it would not work.

Reply:
Fixed in: [307ff39](https://github.com/OpenMined/PySyft/pull/8236/commits/307ff3925bb209dd605b74012e6ce3b55a10acc0)